app/build.gradle.kts

```diff
@@ -91,6 +91,7 @@ dependencies {
     implementation(projects.analytics)
     implementation(projects.config)
     implementation(projects.design)
+    implementation(projects.feature.availability)
```

Comment:
Not sure this needs to be a feature module, feels like more of a function of the app module itself?

Reply:
Yes, I see what you mean. I've got 3 related screens going in (this unavailable one, forced update and recommend update) which is why I thought I'd pop them in a module together. I will move to main app module.

Reply:
Done